pages/restore.tsx

```diff
@@ -51,6 +53,10 @@ const Home: NextPage = () => {
   const [error, setError] = useState<string | null>(null);
   const [photoName, setPhotoName] = useState<string | null>(null);
 
+  // const fetcher = (url: string) => fetch(url).then((res) => res.json());
+  // const { data: remainingMessage } = useSWR("/api/remaining", fetcher);
```

Comment:
Did you mean to leave this in?

Reply:
Yes! It was something I couldn't get working that I just did, gonna push a fix soon. Appreciate you flagging though :)